src/components/HomePage/index.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styles from "./styles.module.css";
+
+export default function Homepage() {
+  return (
+    <>
+      {}
+      <section className={styles.heroSection}>
+        <div className={styles.container}>
+          <div className={styles.content}>
+            <h1 className={styles.title}>
+              WELCOME TO THE TECHNICAL WRITING MENTORSHIP PROGRAM
+            </h1>
+            <a
+              className={styles.button}
+              href="https://discord.gg/your-invite-link"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Join us on Discord
+            </a>
+          </div>
+          <div className={styles.imageContainer}>
+            <img
+              src="https://res.cloudinary.com/dmvphwhvc/image/upload/v1746098734/Untitled_design__3_-removebg-preview_uudx1p.png"
+              alt="Technical writing illustration"
+              className={styles.image}
+            />
+          </div>
+        </div>
+      </section>
+
+      {}
+      <section className={styles.heroSection}>
+        <div className={styles.container}>
+          <div className={styles.imageContainer}>
+            <img
+              src="https://res.cloudinary.com/dmvphwhvc/image/upload/v1746103162/directmentorshipTMWP-removebg-preview_z1edw9.png"
```

Comment:
The color of this illustration is different from the first one.
Use a consistent color shade, and then use high resolution illustrations please.

check https://storyset.com/